deep4cast/forecasters.py

```diff
@@ -94,7 +94,7 @@ def fit(self, ts, lookback_period=10, verbose=2):
         # Change state to fitted so that other methods work correctly.
         self._is_fitted = True
 
-    def predict(self, ts):
+    def predict_point_estimate(self, ts):
```

Comment:
I think this method should be called "predict" because that conforms to the usual naming conventions in Scikit-Learn and other places. In my opinion, it is ideal to have one predict function that can also return samples depending on the input parameters.